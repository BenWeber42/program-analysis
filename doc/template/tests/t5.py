
def f(a, b):
    return 2*a + 3*b
