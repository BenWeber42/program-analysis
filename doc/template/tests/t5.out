5
10
15
0
3
6
4998
5000
